Added image resizing

szul opened the pull request:
Some use cases require a larger signature capture, but a smaller display. These changes allow for passing in a size to resize the image to fit the smaller canvas.

For example, the project I used this in required a 800 x 600 signature capture for touch screens, but displaying that signature as an image in a PDF made it too large, while reducing the canvas size of the SignatureToImage object didn't resize the image, but merely cut it off. This fixes that.

szul committed:
Added image resizing

Some use cases require a larger signature capture, but a smaller
display. These changes allow for passing in a size to resize the image
to fit the smaller canvas.

szul committed:
Fixed whitespace

parrots commented:
Good idea, could you add comments to the alternate constructor and document the params on the new resize method?

szul commented:
No problem.


Michael
________________________________
From: Curtis Herbert<mailto:notifications@github.com>
Sent: ‎4/‎27/‎2015 9:33 PM
To: parrots/SignatureToImageDotNet<mailto:SignatureToImageDotNet@noreply.github.com>
Cc: Michael Szul<mailto:michael@szul.us>
Subject: Re: [SignatureToImageDotNet] Added image resizing (#4)

Good idea, could you add comments to the alternate constructor and document the params on the new resize method?

---
Reply to this email directly or view it on GitHub:
https://github.com/parrots/SignatureToImageDotNet/pull/4#issuecomment-96871136

szul committed:
Documentation and SigNameToImage mods

Added documentation to constructors. Added the same resize mods to the
SigNameToImage method.

szul commented:
Documented. Also made the same modifications to the SigNameToImage method.
